src/js/components/about/AboutUsPage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router';
+import MainBannerSection from './MainBannerSection';
+import WhatWeDo from './WhatWeDo';
+import WhatDrivesUs from './WhatDrivesUs';
+import OurVision from './OurVision';
+// import WhereWeAreNow from './WhereWeAreNow';
+import Leadership from './Leadership';
+// import Partners from './Partners';
+import ContactUsPage from '../contact_us/';
```

Comment:
You may want to confirm that this directory isn't `../contact_us/Index` . Since `Index.js` starts with a capital letter, it wouldn't get resolved automatically by specifying just the directory.

cc @andela-gogbara @andela-morekoya